src/pages/settings/tabs/currencies/edit-currency.vue

```diff
@@ -28,9 +30,21 @@
           :disabled="deletionDisabled"
           @click="onDeleteHandler"
         >
-          Delete
+          Delete currency
         </ui-button>
       </ui-tooltip>
+      <div class="edit-currency__checkbox">
+            <label :for="'checkbox-' + currency.id.toString()" class="edit-currency__label">
+              <span class="live-span">Live update</span>
+              <input
+                :id="'checkbox-' + currency.id.toString()"
+                :checked="!currency.custom"
+                class="tick-field__input"
+                type="checkbox"
+                @change="toggleChange($event)"
```

Comment:
You don't need to explicitly pass `$event` here, just use `@change="toggleChange"`